independent-publisher-connectors/OpenAI GPT-4/apiDefinition.swagger.json

```diff
@@ -312,7 +312,7 @@
                     "gpt-4",
                     "gpt-4-0314",
                     "gpt-4-32k",
-                    "gp-4-32k-0314"
+                    "gpt-4-32k-0314"

```

Comment:
@troystaylor typically this would considered a breaking change and it is not allowed. We will ignore this since this is a new connector. 
![image](https://user-images.githubusercontent.com/97551789/232879236-31cae534-a48b-4fd7-9863-b6488a77f097.png)